common/djangoapps/util/tests/test_codejail_includes.py

```diff
@@ -0,0 +1,24 @@
+# lint-amnesty, pylint: disable=missing-module-docstring
```

Comment:
Instead of silencing this warning, it is best just to add a docstring. It will help others, who may not know what "codejail includes" are, understand the purpose of this module.